features/admin.userstores.v1/constants/user-store-constants.ts

```diff
@@ -196,3 +202,25 @@ export const PRIMARY_USERSTORE: string = "PRIMARY";
  * Character limit for userstore name.
  */
 export const USERSTORE_NAME_CHARACTER_LIMIT: number = 50;
+
+/**
+ * Name of the remote userstore type.
+ */
+export const REMOTE_USERSTORE_TYPE_NAME: string = "WSOutboundUserStoreManager";
```

Comment:
Instead of having two separate constants, can't we use `REMOTE_USER_STORE_TYPES` for the usages of this as well?

Reply:
There is no way to access the `WSOutboundUserStoreManager` from the existing string array. Instead we will have to implement a `map`.